tools/tools.qmd

```diff
@@ -20,8 +20,8 @@ A curated collection of tools and interfaces to help you work effectively with S
 | **Julia**    | [**Stan.jl**](http://stanjulia.github.io/Stan.jl/stable/INTRO/){target="_blank"} | Interface to Stan for Julia users. |
 | **MATLAB**   | [**MatlabStan**](https://github.com/brian-lau/MatlabStan/wiki){target="_blank"} | Interface to Stan for MATLAB users. |
 | **Shell**    | [**CmdStan**](https://mc-stan.org/cmdstan){target="_blank"} | Command-line interface to Stan, usable from any shell environment. |
-| **R**   | [**RStan**](https://mc-stan.org/rstan){target="_blank"} | Legacy R interface to Stan. *(For new projects, consider using CmdStanR.)* |
-| **Python**   | [**PyStan**](https://mc-stan.org/pystan){target="_blank"} | Legacy Python interface to Stan. *(For new projects, consider using CmdStanPy.)* |
+| **R**   | [**RStan**](https://mc-stan.org/rstan){target="_blank"} | R interface to Stan. |
```

Comment:
What about "The Original R interface to Stan". It's not as strong as legacy, but still implies there's a newer option?

Reply:
If you want to I think it could still say "For new projects, consider using CmdStanR". Either way.